language-design.md

```diff
@@ -48,8 +58,11 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 In what way is an API a language? 
 
 **Response**
+APIs can be categorized as internal domain-specific languages, since they are written inside an existing host language.
 
+Most of the APIs are designed with consistency and symmetry, just like any other programming languages since "every API is a little language, and people must learn to read and write it" (Bloch, 2006). This should help create a feeling of fluency that most languages possess. By having this characteristic of languages, APIs make it easy for users to detect patterns to write new code from a similar section.
```

Comment:
I too am unsure what you mean by symmetry. I may be wrong but I feel that you might mean the API itself is "symmetric" as in when using different parts of the API they are similar to their counterparts. Now I am unsure if this differs too much from consistency so perhaps you mean what Ross said and that an API is similar to the user in a similar way to its abstraction? I agree that complex cases the second definition would not hold up.